pa11y/Dockerfile

```diff
@@ -0,0 +1,31 @@
+FROM node:14.5.0-alpine3.10
+
+# Installs latest Chromium (77) package.
+RUN apk add --no-cache \
+      chromium \
+      nss \
+      freetype \
+      freetype-dev \
+      harfbuzz \
+      ca-certificates \
+      ttf-freefont \
+      nodejs \
```

Comment:
I *think* we can remove this as we're already pulling node in from the base image - worth testing locally.

Reply:
Might also be worth looking if there's an official alpine node chromium somewhere (Selenium docker builds might have that for example).

Reply:
I've had a look on Docker but couldn't find any official images that fit the match unfortunately.